src/utils/addressesAreEquivalent.ts

```diff
@@ -1,4 +1,4 @@
-export function addressesAreEquivalent(a?: string, b?: string) {
+export function addressesAreEquivalent(a?: string | null, b?: string | null) {
```

Comment:
nit: I am generally confused with nulls vs undefined in our codebase sometimes, when this can be undefined, and when this can be null?

Reply:
i think this function should allow nullish / undefined values, but it should be required to pass 2 arguments.